src/components/dashboard/sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,37 @@
+import 
+{BsGrid1X2Fill, BsPeopleFill, BsFillGearFill}
+ from 'react-icons/bs'
+ import "../../../pages/DashboardPage.css"
+ import "./Sidebar.css"
+ import logo from "../../../assets/dashboard/logo.svg"
+
```

Comment:
I think its also good practice to add Prop Types when passing props to a component especially in TypeScript. I think right now `openSidebarToggle, OpenSidebar` are underlined in red.
You can insert in this line:
`interface Props {
  OpenSidebar: () => void;
  openSidebarToggle: boolean;
}`